### ISSUE TITLE
↔ [Converter] Add support for aten::clone in TRTorch

### ISSUE THREAD

**Issue by developer0hye** [NVIDIA/TRTorch#342]
##

aten::clone

- **Function Schema**:

`Schema: aten::clone(Tensor self, *, int? memory_format=None) -> (Tensor)`

- **Original PyTorch API**:
`clone(*, memory_format=torch.preserve_format) → Tensor`

- **Relevant TensorRT Documentation**:

**Comment by narendasan** [comment]
Can you provide an example of how you use clone? I am tempted to just duplicate the ITensor reference when you hit clone but i want to make sure that that would still work correctly in real code.

**Comment by developer0hye** [comment]
@narendasan 
Thanks!

The below one is python code.

```python
import torch
import torch.nn as nn

class CloneExample(nn.Module):
    def __init__(self):
        super(CloneExample, self).__init__()
        self.conv = nn.Conv2d(3, 3, 3, 1, 1)
    def forward(self, x):
        identity = x.clone()
        y = torch.relu(self.conv(x))
        return y + identity
    
input = torch.randn((1, 3, 224, 224))
model = CloneExample()
model.eval()
model = torch.jit.script(model)

torch.jit.save(model, 'test.ts')
```

This one is c++ code.
```cpp
#include <iostream>
#include <string>
#include <torch/torch.h>
#include <torch/script.h>
#include <opencv2/opencv.hpp>

#include "trtorch/trtorch.h"

int main()
{
  torch::NoGradGuard no_grad_;
  
  //load model
  torch::jit::script::Module encoder;
  encoder = torch::jit::load("test.ts");
  encoder.eval();
  encoder.to(torch::kCUDA);
  std::cout << "encoder is loaded!" << std::endl;
  
  //convert TS2TR
  auto in = torch::randn({1, 3, 736, 1280}, {torch::kCUDA});
  auto input_sizes = std::vector<trtorch::CompileSpec::InputRange>({in.sizes()});
  trtorch::CompileSpec info(input_sizes);
  info.workspace_size = 1<<30;//1GB
  info.strict_types = true;
  auto trt_encoder = trtorch::CompileGraph(encoder, info);
  return 0;
}
```

When I build c++ code and run, I faced error.

```
encoder is loaded!
ERROR: [TRTorch] - Requested converter for aten::clone, but no such converter was found
terminate called after throwing an instance of 'trtorch::Error'
  what():  [enforce fail at core/conversion/conversion.cpp:113] Expected converter to be true but got false
Unable to convert node: %identity.1 : Tensor = aten::clone(%x.1, %4) # main.py:9:19 (conversion.AddLayer)
Schema: aten::clone(Tensor self, *, int? memory_format=None) -> (Tensor)
Converter for aten::clone requested, but no such converter was found.
If you need a converter for this operator, you can try implementing one yourself
or request a converter: https://www.github.com/NVIDIA/TRTorch/issues

Aborted (core dumped)
```

**Comment by Belval** [comment]
I would also like to see support for `aten::clone` in TRTorch, it's used in some attention models.

**Comment by narendasan** [comment]
Solved by #534
